core/src/test/scala/akka/kafka/scaladsl/IntegrationSpec.scala

```diff
@@ -67,24 +78,35 @@ class IntegrationSpec extends TestKit(ActorSystem("IntegrationSpec"))
     producer.close(60, TimeUnit.SECONDS)
   }
 
+  def produceMessages(topic: String, range: Range) = {
+    Source(range)
```

Comment:
How about if produceMessages returns a Future in order to continue upon completion ?
If so please advise on how to create the Future.